src/main/kotlin/pt/up/fe/ni/website/backend/controller/AuthController.kt

```diff
@@ -28,6 +33,13 @@ class AuthController(val authService: AuthService) {
         return mapOf("access_token" to accessToken)
     }
 
+    @PostMapping("/recoverPassword/{id}")
+    fun generateRecoveryToken(@PathVariable id: Long): Map<String, String> {
+        val recoveryToken = authService.generateRecoveryToken(id)
+        // TODO: Change URL Later
+        return mapOf("recovery_url" to "$backendUrl/accounts/recoverPassword/$recoveryToken")
```

Comment:
I should rename that to `apiUrl` as it better describes my thoughts when implementing it. I didn't consider being a frontend page because we didn't have one, but certainly is the intention when implementing sending emails.

Reply:
Is there any point leaving it with the backend URL? Why not implement the right logic and only send the email in the future issue?

Reply:
Makes sense. Do we already have the frontend page?

Reply:
We don't but just put a config property as a placeholder